include/nbl/video/ILogicalDevice.h

```diff
@@ -334,8 +335,10 @@ class NBL_API2 ILogicalDevice : public core::IReferenceCounted, public IDeviceMe
         // Create an ImageView that can actually be used by shaders (@see ICPUImageView)
         inline core::smart_refctd_ptr<IGPUImageView> createImageView(IGPUImageView::SCreationParams&& params)
         {
-            if (!params.image->wasCreatedBy(this))
+            if (!params.image->wasCreatedBy(this)) {
+                m_logger.log("Invalid image was given [%s - %s:%d]", system::ILogger::ELL_ERROR, __FUNCTION__, __FILE__, __LINE__);
```

Comment:
Improved message:
"The image was not created by this Device"

Reply:
yeah, that's better

Reply:
resolved